src/api/cpp/cvc5.cpp

```diff
@@ -6590,6 +6596,24 @@ Term Solver::getSeparationNilTerm() const
   CVC5_API_TRY_CATCH_END;
 }
 
+Term Solver::declarePool(const std::string& symbol,
```

Comment:
Is `symbol` allowed to be empty? If not, this should be checked.

Reply:
This should be the same policy as `declare-fun`, which I suppose allows the symbol to be emtpy (`||` in SMT-LIB).
Thus, no check is needed.